### PR TITLE
Greater than validation rule.

### PR DESCRIPTION
**Pull request by ghiobi**
Adds a greater than validation rule.

### CODE REVIEW

File: tests/unit/ValidationTest.php [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+class ValidationTest extends TestCase
+{
+    /**
+     * A greater than validation rule test.
+     *
+     * @return void
+     */
+    public function testGreaterThanRule()
+    {
+        $validator = Validator::make([
```
Comment (JTalarico) [inline tests/unit/ValidationTest.php]: Don't they all return void? If so many all the comments should reflect it. Not just this?

File: tests/unit/WalletManagerTest.php [hunk 2]
```diff
@@ -28,21 +37,22 @@ public function testDeposit()
...
         $this->assertNull($wallet);
 
         //3)Withdrawing less than or equal funds than wallet contains
-        $wallet = $walletManager->withdraw(new \Money\Money(200, new \Money\Currency('TEST')));
+        $wallet = $walletManager->withdraw(new Money(200, new Currency('TEST')));
         $balance = $wallet->balance;
+
         $this->assertEquals($balance, 0);
     }
 }
```
Comment (JTalarico) [inline tests/unit/WalletManagerTest.php]: Shouldn't we avoid in -function comments?

File: tests/unit/ValidationTest.php (hunk 1 above)
Comment (JTalarico) [inline tests/unit/ValidationTest.php]: Meant to hi-light line 10-14

File: tests/unit/WalletManagerTest.php (hunk 2 above)
Comment (JTalarico) [inline tests/unit/WalletManagerTest.php]: Meant to hi-light line 43-52

**Comment by ghiobi** [comment]
Correct the inline commenting and added phpdocs.

File: tests/unit/ValidationTest.php (hunk 1 above)
Comment (ghiobi) [inline tests/unit/ValidationTest.php]: Done

**Pull request merged**